scripts/mkcloud

```diff
@@ -1237,6 +1247,11 @@ function sanity_checks()
     if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
+
+    if [[ " ${steplist[*]} " == *" batch "* ]] && [[ "$scenario" == '' ]] ; then
```

Comment:
Thanks for both tips!
But with my approach of checking if one of the steps is `batch` they are necessary as they allow to add a whitespace around the first and last step. I did this to make sure that this pattern doesn't match a hypothetical step like `FOObatchBAR`.
Is there a better way to check if an array contains an element?